js/client/modules/@arangodb/testutils/test-utils.js

```diff
@@ -166,12 +166,17 @@ function filterTestcaseByOptions (testname, options, whichFilter) {
     whichFilter.filter = 'graph';
     return false;
   }
-  
+
   if (testname.indexOf('-nonwindows') !== -1 && platform.substr(0, 3) === 'win') {
     whichFilter.filter = 'non-windows';
     return false;
   }
 
+  if (testname.indexOf('-nonmac') !== -1 && platform.substr(0, 3) === 'win') {
```

Comment:
why does the `nonmac` filter on platforms that are reporting to be Windows?
Should it be `&& platform.substr(0, 6) === 'darwin')` or similar?